deploy/helm/sumologic/conf/setup/custom.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+export SUMOLOGIC_BASE_URL=
+export SUMOLOGIC_ACCESSKEY=
+export SUMOLOGIC_ACCESSID=
+
+# shellcheck disable=SC2010
+for dir in $(ls /customer-scripts | grep _ | grep -oE '^.*?_' | sed 's/_//g' | sort | uniq); do
+  target="/scripts/${dir}"
+  mkdir "${target}"
+  # shellcheck disable=SC2010
+  for file in $(ls "/customer-scripts/${dir}_"* | grep -oE '_.*' | sed 's/_//g'); do
+    cp "/customer-scripts/${dir}_${file}" "${target}/${file}"
+  done
+  cd "${target}" && ls -al && bash setup.sh && cd ..
```

Comment:
`pushd` and `popd` instead of using `cd`s?

Reply:
changed to `cd` and running the script, no need to revert cwd